inspectors/models/abstractinspectormodel.cpp

```diff
@@ -83,6 +84,9 @@ void AbstractInspectorModel::onPropertyValueChanged(const Ms::Pid pid, const QVa
         return;
 
     Ms::Score* score  = parentScore();
+    if (!score)
```

Comment:
In the new style of code, brackets should always be put, without exception (so that there are no exceptions and variations) https://github.com/musescore/Documentation/pull/4